bridge/github/export.go

```diff
@@ -0,0 +1,731 @@
+	m := &addCommentToIssueMutation{}
+	input := githubv4.AddCommentInput{
+		SubjectID: subjectID,
+		Body:      githubv4.String(body),
+	}
+
+	parentCtx := context.Background()
+	ctx, cancel := context.WithTimeout(parentCtx, defaultTimeout)
+	defer cancel()
+
+	if err := gc.Mutate(ctx, m, input, nil); err != nil {
+		return "", "", err
+	}
+
+	node := m.AddComment.CommentEdge.Node
+	return node.ID, node.URL, nil
+}
+
+func editCommentGithubIssue(gc *githubv4.Client, commentID, body string) (string, string, error) {
+	m := &updateIssueCommentMutation{}
+	input := githubv4.UpdateIssueCommentInput{
+		ID:   commentID,
+		Body: githubv4.String(body),
+	}
+
+	parentCtx := context.Background()
+	ctx, cancel := context.WithTimeout(parentCtx, defaultTimeout)
+	defer cancel()
+
+	if err := gc.Mutate(ctx, m, input, nil); err != nil {
+		return "", "", err
+	}
+
+	return commentID, m.UpdateIssueComment.IssueComment.URL, nil
+}
+
+func updateGithubIssueStatus(gc *githubv4.Client, id string, status bug.Status) error {
+	m := &updateIssueMutation{}
+
+	// set state
+	state := githubv4.IssueStateClosed
+	if status == bug.OpenStatus {
+		state = githubv4.IssueStateOpen
+	}
+
+	input := githubv4.UpdateIssueInput{
+		ID:    id,
+		State: &state,
+	}
+
+	parentCtx := context.Background()
+	ctx, cancel := context.WithTimeout(parentCtx, defaultTimeout)
+	defer cancel()
+
+	if err := gc.Mutate(ctx, m, input, nil); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func updateGithubIssueBody(gc *githubv4.Client, id string, body string) error {
+	m := &updateIssueMutation{}
+	input := githubv4.UpdateIssueInput{
+		ID:   id,
+		Body: (*githubv4.String)(&body),
+	}
+
+	parentCtx := context.Background()
+	ctx, cancel := context.WithTimeout(parentCtx, defaultTimeout)
+	defer cancel()
+
+	if err := gc.Mutate(ctx, m, input, nil); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func updateGithubIssueTitle(gc *githubv4.Client, id, title string) error {
+	m := &updateIssueMutation{}
+	input := githubv4.UpdateIssueInput{
+		ID:    id,
+		Title: (*githubv4.String)(&title),
+	}
+
+	parentCtx := context.Background()
+	ctx, cancel := context.WithTimeout(parentCtx, defaultTimeout)
+	defer cancel()
+
+	if err := gc.Mutate(ctx, m, input, nil); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// update github issue labels
+func (ge *githubExporter) updateGithubIssueLabels(gc *githubv4.Client, labelableID string, added, removed []bug.Label) error {
+	addedIDs, err := ge.getLabelsIDs(gc, labelableID, added)
+	if err != nil {
+		return errors.Wrap(err, "getting added labels ids")
+	}
+
+	m := &addLabelsToLabelableMutation{}
+	inputAdd := githubv4.AddLabelsToLabelableInput{
+		LabelableID: labelableID,
+		LabelIDs:    addedIDs,
+	}
+
+	parentCtx := context.Background()
+	ctx, cancel := context.WithTimeout(parentCtx, defaultTimeout)
+
+	// add labels
+	if err := gc.Mutate(ctx, m, inputAdd, nil); err != nil {
+		cancel()
+		return err
+	}
+	cancel()
+
+	if len(removed) > 0 {
```

Comment:
if len(removed) == 0, you can return at the very beginning of this function

Reply:
Done